### PR TITLE
feat: support closeable option on progress api

### PR DESCRIPTION
**Pull request by erha19**
### Types

- [x] 🎉 New Features

### Background or solution

Usage:

```ts
this.progressService.withProgress({
    location: ProgressLocation.Notification,
    title: 'test',
    closeable: false,
 }, () => {}, () => {
     // cancel
});
```

image

### Changelog

support closeable option on progress api

### CODE REVIEW

File: packages/core-browser/src/progress/progress.service.tsx [hunk 1]
```diff
@@ -303,6 +304,7 @@ export class ProgressService implements IProgressService {
 
     const createNotification = (message: string, silent: boolean, increment?: number): string => {
       const buttons: string[] = [];
+      const closeable = isDefined(options.closeable) ? options.closeable : true;
```
Comment (hacke2) [inline packages/core-browser/src/progress/progress.service.tsx:307]: options.closeable ?? true

Comment (erha19) [reply packages/core-browser/src/progress/progress.service.tsx:307]: 这样写就是 always true 了

Comment (hacke2) [reply packages/core-browser/src/progress/progress.service.tsx:307]: options.closeable || true 才是  always true

**Review by hacke2 — APPROVED** [review]
+1

**Pull request merged**